test/dynamo/test_higher_order_ops.py

```diff
@@ -3751,6 +3824,395 @@ def wrapper_fn(x, y):
         self.assertEqual(len(counters["graph_break"]), 0)
         self.assertEqual(actual, expected)
 
+    @config.patch(capture_func_transforms=True)
+    def test_jvp_simple(self):
+        if check_dynamic_shape_capture():
+            self.skipTest("test fails with dynamic shapes")
+
+        counters.clear()
+
+        def fn(x):
+            return x.sin().sum()
+
+        def wrapper_fn(x, v):
+            return torch.func.jvp(fn, (x,), (v,))
+
+        x = torch.randn(3, 3)
+        v = torch.randn(3, 3)
+        wrapped_gm = self._compile_check(wrapper_fn, (x, v))
+
+        actual = normalize_gm(wrapped_gm.print_readable(print_output=False))
+        self.assertExpectedInline(
+            actual,
+            """\
+class GraphModule(torch.nn.Module):
+    def forward(self, L_x_ : torch.Tensor, L_v_ : torch.Tensor):
+        p = L_x_
+        t = L_v_
+
+        _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable("torch.func transforms don't yet support saved tensor hooks. Please open an issue with your use case.")
+
+        _jvp_treespec_compare = torch._functorch.eager_transforms._jvp_treespec_compare((p,), (t,))
+
+        _jvp_increment_nesting = torch._C._functorch._jvp_increment_nesting()
+        _set_fwd_grad_enabled = torch._C._set_fwd_grad_enabled(True)
+
+        count_jvp_interpreters = torch._C._functorch.count_jvp_interpreters()
+
+        _enter_dual_level = torch._C._enter_dual_level()
+
+        maybe_load_decompositions = torch.autograd.forward_ad.maybe_load_decompositions()
+
+        _make_dual = torch._make_dual(p, t, level = 0);  p = t = None
```

Comment:
What is `_make_dual` when tracing? it should be a GradTrackingTensor that has a tangent on it (one can tell if it has a tangent by calling unpack_dual on it)

Reply:
The following test case works with torch.compile:

```python
import torch
fwAD = torch.autograd.forward_ad

@torch.compile(backend='eager', fullgraph=True)
def fn(x, y):
    with fwAD.dual_level():
        dual = fwAD.make_dual(x, y)
        return fwAD.unpack_dual(dual)

x = torch.tensor([1.23])
y = torch.tensor([4.56])
y = fn(x, y)
print(y)
# UnpackedDualTensor(primal=tensor([1.2300]), tangent=tensor([4.5600]))
```

And the generated FX graph:

```python
GraphModule(torch.nn.Module):
    def forward(self, L_x_ : torch.Tensor, L_y_ : torch.Tensor):
        l_x_ = L_x_
        l_y_ = L_y_

        # No stacktrace found for following nodes
        _enter_dual_level = torch._C._enter_dual_level()

        # File: /home/guilhermeleobas/git/pytorch/torch/autograd/forward_ad.py:108 in make_dual, code: maybe_load_decompositions()
        maybe_load_decompositions = torch.autograd.forward_ad.maybe_load_decompositions()

        # File: /home/guilhermeleobas/git/pytorch/torch/autograd/forward_ad.py:127 in make_dual, code: return torch._VF._make_dual(tensor, tangent, level=level)
        dual = torch._make_dual(l_x_, l_y_, level = 0);  l_x_ = l_y_ = None

        # File: /home/guilhermeleobas/git/pytorch/torch/autograd/forward_ad.py:170 in unpack_dual, code: primal, dual = torch._VF._unpack_dual(tensor, level=level)
        _unpack_dual = torch._unpack_dual(dual, level = 0);  dual = None
        primal = _unpack_dual[0]
        dual_1 = _unpack_dual[1];  _unpack_dual = None

        # No stacktrace found for following nodes
        _exit_dual_level = torch._C._exit_dual_level(0)
        return (primal, dual_1)
```